daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/DPath.scala

```diff
@@ -160,6 +161,12 @@ final class RuntimeExpressionDPath[T <: AnyRef](qn: NamedQName, tt: NodeInfo.Kin
             case _ => doSDE(e, ps)
           }
         }
+        case ParserNonBlocking => {
+          e match {
+            case nd: InfosetNoDataExceptionBase => doSDE(new InfosetSelfReferencingException(state.dState.currentNode.asElement, state.dState.currentNode.erd), ps)
```

Comment:
This line is a bit too long. Can you split it into multiple lines:
```suggestion
            case nd: InfosetNoDataExceptionBase => {
              val sre = new InfosetSelfReferencingException(state.dState.currentNode.asElement, state.dState.currentNode.erd)
              doSDE(sre, ps)
            }
```

Reply:
Done